src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -2387,23 +2842,32 @@ await using (var context = InMemoryContentDbContext(contextId))
             {
                 var service = BuildThemeService(context);
 
-                var result = await service.GetPublicationTree(PublicationTreeFilter.LatestData);
+                var result = await service.GetPublicationTree(PublicationTreeFilter.DataCatalogue);
 
                 Assert.Single(result);
                 Assert.Equal("Theme A", result[0].Title);
+                Assert.Equal("Theme A summary", result[0].Summary);
 
                 Assert.Single(result[0].Topics);
                 Assert.Equal("Topic A", result[0].Topics[0].Title);
 
                 var publications = result[0].Topics[0].Publications;
 
-                Assert.Single(publications);
+                Assert.Equal(2, publications.Count);
+                Assert.Equal("publication-a", publications[0].Slug);
                 Assert.Equal("Publication A", publications[0].Title);
+                Assert.Equal(PublicationType.NationalAndOfficial, publications[0].Type);
+                Assert.Null(publications[0].LegacyPublicationUrl);
+
+                Assert.Equal("publication-b", publications[1].Slug);
+                Assert.Equal("Publication B", publications[1].Title);
+                Assert.Equal(PublicationType.NationalAndOfficial, publications[1].Type);
+                Assert.Null(publications[1].LegacyPublicationUrl);
             }
         }
 
         [Fact]
-        public async Task GetPublicationTree_LatestData_DoesNotFilterPublicationWithDataOnLatestRelease()
+        public async Task GetPublicationTree_DataCatalogue_FiltersPublicationWithNoDataOnLatestAmendedRelease()
```

Comment:
Seems like this is test case would assert the same thing as another test case above: `GetPublicationTree_DataCatalogue_FiltersPublicationWithNoDataOnAmendedLatestRelease`

Reply:
They appear to be doing the same thing so I've removed one